cypress/e2e/connector/connector.cy.js

```diff
@@ -0,0 +1,81 @@
+describe("connector", () => {
+  const username = `cypress${Math.round(+new Date() / 1000)}@gmail.com`;
+  const password = "Cypress98#";
+
+  // Login before each testcase
+  beforeEach(() => {
+    cy.visit("http://localhost:9000/dashboard/login");
+    cy.url().should("include", "/login");
+    cy.get("#card-header", { timeout: 10000 }).should(
+      "contain",
+      "Hey there, Welcome back!",
+    );
+    cy.get("#card-subtitle")
+      .should("contain", "Sign up")
+      .click({ force: true });
+    cy.url().should("include", "/register");
+    cy.get("#auth-submit-btn").should("exist");
+    cy.get("#tc-text").should("exist");
+    cy.get("#footer").should("exist");
+
+    cy.get("[data-testid=email]").type(username);
+    cy.get("[data-testid=password]").type(password);
+    cy.get('button[type="submit"]').click({ force: true });
+    cy.get("[data-testid=skip-now]", { timeout: 3000 }).click({ force: true });
+
+    cy.wait(3000);
+    cy.url().should("include", "/dashboard/home");
+  });
+
+  it("Create a dummy connector", () => {
+    cy.url().should("include", "/dashboard/home");
+
+    cy.get('[data-form-label="Business name"]', { timeout: 15000 }).should(
+      "exist",
+    );
+    cy.get("[data-testid=merchant_name]", { timeout: 10000 }).type(
```

Comment:
Is the addition of  timeout necessary?

Reply:
no it is not removed it!

Reply:
The commit message is not explanatory. Please resolve the issue properly.

Reply:
Hey @kanikabansal-juspay , initially i have put additional time out for testing purpose, now i have tested them by keeping 10000, and it is working

Reply:
The timeout is not required in all the places since the API responds promptly. You can safely remove it, and the test will still work as expected. Once you've made the changes, you can push them.

Reply:
Understood will remove them and test and push, can i squash commits and push?

Reply:
yes you can squash commits and push and please don't resolve the conversation.

Reply:
Sure won't resolve them

Reply:
Cypress tests check is failing in checkout code itself, I am not sure why this is happening, can you please help me out ?

Reply:
please ignore those tests

Reply:
removed timeouts please review in your free time!